src/main/java/dev/symphony/harmony/mixin/enchanting/GrindstoneRepairsWithMaterials.java

```diff
@@ -0,0 +1,70 @@
+package dev.symphony.harmony.mixin.enchanting;
+
+import com.llamalad7.mixinextras.injector.wrapoperation.Operation;
+import com.llamalad7.mixinextras.injector.wrapoperation.WrapOperation;
+import dev.symphony.harmony.config.HarmonyConfig;
+import net.minecraft.inventory.Inventory;
+import net.minecraft.item.ItemStack;
+import net.minecraft.screen.GrindstoneScreenHandler;
+import net.minecraft.screen.slot.Slot;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+// FEATURE: Allowed grindstones to use base materials to repair
+// AUTHORS: Randomvideos
```

Comment:
Use javadocs instead of comments please! You can take a look at HuskEntityMixin (in mobs package) for an example.